packages/core/src/media/styling/container.ts

```diff
@@ -0,0 +1,159 @@
+        },
+      },
+      [aspectRatios[1].key]: {
+        aspectRatio: aspectRatios[1].value,
+
+        [`@supports not (aspect-ratio: ${aspectRatios[1].value})`]: {
+          '&:before': {
+            float: 'left',
+            paddingTop: `calc(${aspectRatios[1].value})%`,
+            content: '',
+          },
+
+          '&:after': {
+            display: 'block',
+            content: '',
+            clear: 'both',
+          },
+        },
+      },
+      [aspectRatios[2].key]: {
+        aspectRatio: aspectRatios[2].value,
+
+        [`@supports not (aspect-ratio: ${aspectRatios[2].value})`]: {
+          '&:before': {
+            float: 'left',
+            paddingTop: `calc(${aspectRatios[2].value})%`,
+            content: '',
+          },
+
+          '&:after': {
+            display: 'block',
+            content: '',
+            clear: 'both',
+          },
+        },
+      },
+      [aspectRatios[3].key]: {
+        aspectRatio: aspectRatios[3].value,
+
+        [`@supports not (aspect-ratio: ${aspectRatios[3].value})`]: {
+          '&:before': {
+            float: 'left',
+            paddingTop: `calc(${aspectRatios[3].value})%`,
+            content: '',
+          },
+
+          '&:after': {
+            display: 'block',
+            content: '',
+            clear: 'both',
+          },
+        },
+      },
+      [aspectRatios[4].key]: {
+        aspectRatio: aspectRatios[4].value,
+
+        [`@supports not (aspect-ratio: ${aspectRatios[4].value})`]: {
+          '&:before': {
+            float: 'left',
+            paddingTop: `calc(${aspectRatios[4].value})%`,
+            content: '',
+          },
+
+          '&:after': {
+            display: 'block',
+            content: '',
+            clear: 'both',
+          },
+        },
+      },
+    },
+  },
+});
+
+const fullscreen = keyframes({
+  '0%': { opacity: 0 },
+  '100%': { opacity: 1 },
+});
+
+export const container = css(aspectRatioContainer, {
+  fontFamily:
+    '$display, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji',
+  backgroundColor: '$background',
+  overflow: 'hidden',
+  maxWidth: '100%',
+  width: '100%',
+  height: '100%',
+  maxHeight: '100%',
+
+  position: 'relative',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+
+  svg: {
+    pointerEvents: 'none',
+  },
+
+  // for iOS rendering custom controls
+  zIndex: 2147483647,
+
+  variants: {
+    size: {
+      fullscreen: {
+        animation: `${fullscreen} 0.2s`,
+      },
+      default: {
+        '&:hover': {
+          boxShadow: '$containerShadowHover',
+        },
+
+        boxShadow: '$containerShadow',
+
+        borderStyle: '$containerBorderStyle',
+        borderColor: '$containerBorderColor',
+        borderRadius: '$containerBorderRadius',
+        borderWidth: '$containerBorderWidth',
```

Comment:
Let's remove the player's default border styles `borderStyle`, `borderColor`, `borderRadius`, `borderWidth`. I don't think the player should have a border - looks better without IMO :)